storage/storage.cpp

```diff
@@ -424,7 +420,7 @@ LocalFilePtr Storage::GetLatestLocalFile(CountryFile const & countryFile) const
 
 LocalFilePtr Storage::GetLatestLocalFile(CountryId const & countryId) const
 {
-  CHECK_THREAD_CHECKER(m_threadChecker, ());
+  //CHECK_THREAD_CHECKER(m_threadChecker, ());
```

Comment:
? An explicit comment that it may not be called on UI thread can be helpful.

Reply:
This CHECK comment (otherwise I'd delete it) is the descriptive comment here and above.